### PR TITLE
Color Schemes: Import Muriel Color Palette from Color Studio

### PR DESCRIPTION
**Pull request by frontdevde**
#### Changes proposed in this Pull Request

*  Import Muriel Color Palette from Color Studio
* For more info see: https://github.com/Automattic/color-studio/

#### Testing instructions

* Build the styles, everything should work as expected.
* As a test `$white` has `$muriel-white` assigned. All occurences of `$white` should still be white ;)

### CODE REVIEW

File: assets/stylesheets/shared/_colors.scss [hunk 1]
```diff
@@ -1,65 +1,73 @@
+/** @format */
+
+// The Muriel Color Palette
+@import '../../../node_modules/color-studio/dist/colors';
```
Comment (flootr) [inline assets/stylesheets/shared/_colors.scss]: I think the sass-loader can do something like `~color-studio/dist/colors`. Let's also use an explicit extension for the file we load.

Comment (porada) [reply assets/stylesheets/shared/_colors.scss]: That stylesheet is referenced in `package.json`, so `color-studio` should also work in that case. 🙂

Comment (frontdevde) [reply assets/stylesheets/shared/_colors.scss]: @flootr @porada As per documentation, both methods should work, yes. In practice though, they did error out for me though when testing them locally previously. 🤔Feel free to test it out on this branch, good chance that I'm probably missing something. :)

Comment (flootr) [reply assets/stylesheets/shared/_colors.scss]: The reason this doesn't work is that our `_colors.scss` is not processed with Webpack so we can't use this resolve mechanism from `sass-loader` (yet). Sorry, I didn't see this when I first looked at it.

**Pull request merged**